app/src/main/kotlin/github/com/rhacco/dota2androidapp/viewmodel/MatchesViewModel.kt

```diff
@@ -56,24 +62,19 @@ class MatchesViewModel(application: Application) : AndroidViewModel(application)
                                 newItemData.mTitle = App.instance.getString(
                                         R.string.heading_live_ranked_match, averageMMR, result.match.matchid)
                             if (result.teams?.size == 2) {
```

Comment:
Is there ever a time when the # of teams is < 2?

Reply:
*Probably* no, but a quick safety check should never be bad for sustainable code. A remote API might malfunction or change over time which we have no control over, these few extra lines can help a lot